scripts/bin/update-packages

```diff
@@ -206,6 +291,8 @@ if [[ "$1" == "@all" ]]; then
 			# and not care about anything else in between. If something fails to update,
 			# it will be reported by failure handling code, so no worries.
 			_run_update "${pkg_dir}"
+			echo "termux - took $(( $(date +%s%N | cut -b1-13) - $_D )) ms"
```

Comment:
[`$` prefix on variables is unneeded in arithmetic context](https://www.shellcheck.net/wiki/SC2004).
```suggestion
			echo "termux - took $(( $(date +%s%N | cut -b1-13) - _unix_milli )) ms"
```